util/microservice-sdk/js/examples/consumer/sample.js

```diff
@@ -0,0 +1,25 @@
+const { logger } = require('@dojot/dojot-module-logger');
+const { Kafka: { Consumer } } = require('../index.js');
+
+const consumer = new Consumer({
+  kafka: {
+    'group.id': process.env.KAFKA_GROUP_ID || 'sdk-consumer-example',
+    'metadata.broker.list': process.env.KAFKA_HOSTS || 'localhost:9092',
+    'auto.offset.reset': 'beginning',
+  },
+});
+
+const TAG = { filename: 'sample' };
+
+consumer.init().then(() => {
+  // the target kafka topic, it could be a String or a RegExp
+  const targetTopic = process.env.KAFKA_TOPIC || 'consumer.testing';
+
+  // Register callback to process incoming device data
+  /* const idCallback = */ consumer.registerCallback(targetTopic, (data) => {
```

Comment:
Checking the registerCallback function I realize that it's possible to register more than 1 callback for the topic, that is the behavior we are expecting?

Reply:
That's the expected behavior.

Reply:
Ok, I understand.